packages/core/stories/progress/progress.qa.stories.tsx

```diff
@@ -35,6 +37,11 @@ export const ExamplesGrid: StoryFn<QAContainerProps> = (props) => {
         hideLabel
       />
       <CircularProgress aria-label="Download" value={38} hideLabel />
+      <LinearProgress
+        aria-label="Download"
+        className="noAnimation"
```

Comment:
Is this actually needed? Chromatic should already pause animations https://www.chromatic.com/docs/animations/#css-animations

Reply:
Doesn't work without